qemu/tests/block_check_memory_leak.py

```diff
@@ -50,6 +51,11 @@ def _execute_io_in_guest():
 
     timeout = params.get_numeric('login_timeout', 360)
     session = vm.wait_for_login(timeout=timeout)
+    if arch.ARCH in ('ppc64', 'ppc64le'):
+        cmd_output = vm.monitor.info("pic", False)
+        if "irqchip: in-kernel" in cmd_output:
+            test.cancel("Skip test for xive kvm interrupt guest due to"
```

Comment:
Do we need to check if it's a XICS guest?

Reply:
No,just need to check XIVE guest, have updated.